benchmarks/benchmark/tools/profile-generator/container/benchmark_serving.py

```diff
@@ -404,17 +438,41 @@ def print_metrics(metrics: Dict[str, Optional[str]], duration: float, backend: s
       # handle response
       if request_post.ok:
         if response["status"] == "success":
-          metric_results[query_name] = response["data"]["result"][0]["value"][1]
+          metric_results[query_name] = float(response["data"]["result"][0]["value"][1])
           print("%s: %s" % (query_name, response["data"]["result"][0]["value"][1]))
         else:
           print("Cloud Monitoring PromQL Error: %s" % (response["error"]))
       else:
         print("HTTP Error: %s" % (response))
-    all_metric_results["metric_names"][metric] = metric_results
-    if metric_alias is not None:
-      all_metric_results["metric_aliases"][metric_alias] = metric_results
-  return all_metric_results
-
+    server_metrics[metric] = metric_results
+  return server_metrics
+
+def get_stats_for_set(name, description, points):
+  avg = np.mean(points)
+  median = np.median(points)
+  sd = np.std(points)
+  min = np.min(points)
+  max = np.max(points)
+  p90 = np.percentile(points, 90)
+  p99 = np.percentile(points, 99)
+
+  print(f"Average {description}:" f" {avg:.2f}")
```

Comment:
Since we are adding all the p90, mean, sd data in the summary stat, can we keep this section light and report only mean like before? Other option is to club them into separate sections like we do in summary stats so it is more readable. wdyt?

Reply:
Sounds good. Even in json it would be good to not repeat it in two places. So, thanks for addressing this.